modules/prompt/functions/async

```diff
@@ -1 +1 @@
-../external/pure/async.zsh
\ No newline at end of file
+../external/async/async.zsh
```

Comment:
Uh, that's actually the wrong symlink. It should be pointing to `../external/async/async.zsh`

My guess is that you didn't update submodules, so it wasn't pointing to the right place.